src/cmd/cmd.zig

```diff
@@ -89,6 +90,11 @@ fn execute(_: []const []const u8) !void {
         .{},
     );
 
+    if (build_options.disable_tracing and config.enable_trace) {
+        std.log.err("Tracing is disabled in this build.\n", .{});
+        return;
```

Comment:
better returning an error here no ?

Reply:
Maybe?

I mean, the user probably wants to be properly warned about what's the problem. It's not like it's an obscure error. It's a user invalid usage I'd say.